app/models/card.rb

```diff
@@ -1,21 +1,17 @@
 class Card < ApplicationRecord
-  require "date"
-
   validates :original_text, :translated_text, presence: true
   validate :coincidence, on: [ :create, :update ]
 
   before_save :date_change, on: [ :create, :update ]
 
   private
     def date_change
-      self.review_date = Date.today + 3
+      self.review_date = 3.days.ago
```

Comment:
3.days.ago — это 3 дня назад, а вот 3.days.since —  это через 3 дня 

http://api.rubyonrails.org/classes/Date.html этот и другие методы, которые добавлены в Date в Rails https://ruby-doc.org/stdlib-2.1.1/libdoc/date/rdoc/Date.html — дефолтный Date